### ISSUE TITLE
Export Documentation to static HTML

### ISSUE THREAD

**Issue by scottgutman** [koumoul-dev/vue-openapi#11]
Is there a way to export the documentation to static files?

**Comment by nicolas-bonnel** [comment]
I don't think it is possible with vue-material framework. We are now using vuetify in other projects and are thinking about migrating this component to this framework. Vuetify can be used with Nuxt that can be used to generate static HTML.

**Comment by audetcameron** [comment]
@scottgutman you could use headless chrome to render the html via puppeteer (npm and node based)

**Comment by BatLeDev** [comment]
This component is deprecated and has been superseded by data-fair/openapi-viewer, a standalone Vue 3 service.

Static-HTML export was out of scope for this runtime Vue component; if you need it, please raise it on the new project. Closing here as the repo is no longer developed.
